.gitignore

```diff
@@ -24,6 +24,7 @@ android-template/app/app.iml
 .project
 .env
 capacitor-build.log
+yarn.lock
```

Comment:
We don't want to add yarn.lock to the the template in this PR, we'd want to look at that seperately.

Reply:
Thanks @markemer I have reverted the commit